src/inmanta/agent/config.py

```diff
@@ -174,6 +174,16 @@
     is_lower_bounded_int(60),
 )
 
+scheduler_resource_compliance_check_window = Option(
```

Comment:
If we reuse the old ones I will have to add support for the case a cron schedule is passed but I think it's doable.

Reply:
That's a fair point. The cron expressions sort of go against the grain of what we're trying to express here. I'd pass this by @wouterdb.

Reply:
Otoh, the cron expressions are a very important feature. For operations, to have all repairs at night is very important. 

So we can't just drop that.

Reply:
For the cron expression we probably don't want to make it per-resource btw (as long as the setting is global at least). Otherwise we'd have *a lot* of per-resource deploys/repairs all at the same time, all potentially pulling in dependencies. It makes a lot more sense to consider it a global (environment-wide) trigger then.